src/SimpleSAML/Metadata/MetaDataStorageSource.php

```diff
@@ -349,9 +349,8 @@ private function getDynamicHostedUrl(string $set): string
 
 
     /**
-     * Updates the metadata entry's entity id and returns the modified array.  If the entity id is __DYNAMIC:*__ a
-     * the current url is assigned.  If it is explicit the entityid array key is updated to the entityId that was
-     * provided.
+     * Updates the metadata entry's entity id and returns the modified array.
+     * If it is explicit the entityid array key is updated to the entityId that was provided.
```

Comment:
It is now always explicit.

Is this method still useful now, since it doesn't do much special?